parser/sql/dialect/mysql/src/main/antlr4/imports/mysql/DDLStatement.g4

```diff
@@ -189,8 +189,7 @@ dropTable
     ;
 
 dropIndex
-    : DROP INDEX indexName (ON tableName)?
-    (alterAlgorithmOption | alterLockOption)*
+    : DROP INDEX indexName (ON tableName)? alterAlgorithmOption? alterLockOption?
```

Comment:
Please refer to MySQL syntax.
https://github.com/mysql/mysql-server/blob/1bfe02bdad6604d54913c62614bde57a055c8332/sql/sql_yacc.yy#L9089